lib/stitches/generator_files/app/controllers/api/api_controller.rb

```diff
@@ -1,10 +1,20 @@
 class Api::ApiController < ActionController::Base
   include Stitches::Deprecation
+  #
+  # The order of the rescue_from blocks is important - ActiveRecord::RecordNotFound must come after StandardError,
+  # otherwise "not_found" errors will get rescued in the StandardError block.
+  # See the documentation for rescue_from for further explanation:
+  # https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from
+  # Specifically, this part: "Handlers are inherited. They are searched from right to left, from bottom to top, and up
+  # the hierarchy."
+  #
+  rescue_from StandardError do |exception|
+    Bugsnag.notify(exception)
```

Comment:
Given that this is open source and perhaps not everyone who uses this gem may be using bugsnag, could/should we do something like `Bugsnag.notify(exception) if defined?(Bugsnag)`?

Reply:
Given that this file is generated, I think we should leave Bugsnag out of this and we can have our app template insert it into new apps. Kind of a PITA, I know, but I think that's the least surprising for non-Stitch Fixers

Reply:
This is a good callout! I was definitely not thinking about cases outside Stitch Fix. 😄

It doesn't seem like too much trouble to insert this line using the `app-temlate`, so I'll try that!